app/presenters/hyku/manifest_enabled_work_show_presenter.rb

```diff
@@ -58,5 +63,30 @@ def query_for_rendering(file_set_id)
                                         fl: [ActiveFedora.id_field, 'label_ssi', 'mime_type_ssi'],
                                         rows: 1)
       end
+
+      # Retrieve the required fields from the Form class. Derive the Form class name from the Model name.
+      #   The class should exist, but it may not if different amespaces are being used.
+      #   If it does not exist, rescue and return an empty Array.
+      #
+      # @return [Array] required fields
+      def metadata_fields
+        ns = "Hyrax"
+        model = solr_document['has_model_ssim'].first
+
+        "#{ns}::#{model}Form".constantize.required_fields
+      rescue
+        []
+      end
+
+      # Get the metadata value; return a String for single values and an Array of hashes for multiple values.
+      #
+      # @return [String or Array] field value(s)
+      def get_metadata_value(field)
+        if send(field).count == 1
+          send(field).first
+        else
+          send(field).collect { |f| f }
```

Comment:
I don't think it hurts for the value to be an array of a single string so it might just be easiest to do 
```
def get_metadata_value(field)
  Array.wrap(send(field))
end
```

Reply:
good point, done!